src/timeMachine/components/view_options/ScatterOptions.tsx

```diff
@@ -175,6 +176,11 @@ const ScatterOptions: FunctionComponent<Props> = props => {
         availableColumns={numericColumns}
         axisName="x"
       />
+      {xColumn === '_time' && (
```

Comment:
I just searched around and noticed that we don't have any kind of typed description of `'_time'` which is odd, since it's so important.

https://github.com/influxdata/ui/issues/499